README.md

```diff
@@ -85,6 +85,23 @@ jobs:
 ```
 
 ## Publishing using Apache Maven
+
+### Extra setup for pom.xml:
+
+Maven GPG Plugin configuration in pom.xml file should contain the following structure to avoid possible issues like `Inappropriate ioctl for device` or `gpg: signing failed: No such file or directory`:
```

Comment:
```suggestion
The Maven GPG Plugin configuration in the pom.xml file should contain the following structure to avoid possible issues like `Inappropriate ioctl for device` or `gpg: signing failed: No such file or directory`:
```

Reply:
Done. Thank you.